src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -175,7 +176,15 @@ std::shared_ptr<Node> u4_compression_stack(const OutputVector& list_elems, int64
 
     if (list_elems.size() != 2)
         return nullptr;
-    auto bitwise_and = cast_fw_node(list_elems[0].get_node_shared_ptr(), "aten::bitwise_and");
+
+    auto bitwise_and_candidate = list_elems[0].get_node_shared_ptr();
+    std::shared_ptr<Node> bitwise_and = cast_fw_node(bitwise_and_candidate, "aten::bitwise_and");
+    if (!bitwise_and) {
+        bitwise_and = std::dynamic_pointer_cast<v13::BitwiseAnd>(bitwise_and_candidate);
+        if (!bitwise_and)
+            return nullptr;
```

Comment:
these two lines repeats a behavior of lines 188-189, could we skip them, then?

Reply:
Ah, good catch, thanks.